src/app/forms/templates/trl/trl-purcheser.template.ts

```diff
@@ -0,0 +1,70 @@
+import { FormNode, FormNodeTypes, FormNodeViewTypes } from '../../services/dynamic-form.types';
+
+export const TrlPurchasers: FormNode = {
+  name: 'purchaserDetails',
+  type: FormNodeTypes.GROUP,
+  label: 'Purchasers',
+  viewType: FormNodeViewTypes.SUBHEADING,
+  children: [
+      {
+        name: 'name',
+        label: 'Name',
+        value: '',
+        type: FormNodeTypes.CONTROL
+      },
+      {
+        name: 'address',
+        label: 'Building and street (box 1 of 2)',
+        value: '',
+        type: FormNodeTypes.CONTROL
+      },
+      {
+        name: 'address2',
+        label: 'Building and street (box 2 of 2)',
+        value: '',
+        type: FormNodeTypes.CONTROL
+      },
+      {
+        name: 'postTown',
+        label: 'Town or city',
+        value: '',
+        type: FormNodeTypes.CONTROL
+      },
+      {
+        name: 'address3',
+        label: 'County',
+        value: '',
+        type: FormNodeTypes.CONTROL
+      },
+      {
+        name: 'postCode',
+        label: 'Postcode',
+        value: '',
+        type: FormNodeTypes.CONTROL
+      },
+      {
+        name: 'telephonenumber',
```

Comment:
Is there a reason this is all lowercase or is that just how it is in the database?

Reply:
Just how it is im the spec folder, can double check in dynamo swell